pkg/master/master.go

```diff
@@ -231,21 +231,15 @@ func (m *Master) InstallAPIs(c *Config) {
 
 	// Install extensions unless disabled.
 	if c.APIResourceConfigSource.AnyResourcesForVersionEnabled(extensionsapiv1beta1.SchemeGroupVersion) {
-		m.thirdPartyStorage = c.StorageDestinations.APIGroups[extensions.GroupName].Default
+		var err error
```

Comment:
> Dont need this line?

`:=` below will fail compilation because `m.thirdPartyStorage` is not new.  Have to pre-declare the error.